Source/WebInspectorUI/UserInterface/Views/InlineSwatch.js

```diff
@@ -251,6 +268,22 @@ WI.InlineSwatch = class InlineSwatch extends WI.Object
                 return;
             }
 
+            if (this._subType === WI.InlineSwatch.Type.Color) {
+                let resolvedColor = this._colorPalette.find(palette => value.toString().slice(4, value.length - 1) == palette.colorSource || value.toString() === palette.colorSource).color;
```

Comment:
style: we always wrap the parameter of an arrow function in `(` `)`
style: we always use `===`